asciidoctorj-core/src/main/java/org/asciidoctor/ast/InlineImpl.java

```diff
@@ -0,0 +1,33 @@
+package org.asciidoctor.ast;
+
+import org.jruby.Ruby;
+
+public class InlineImpl extends AbstractNodeImpl implements Inline  {
+
+    protected Inline delegate;
+
+    public InlineImpl(Inline delegate, Ruby ruby) {
+        super(delegate, ruby);
+        this.delegate = delegate;
+    }
+
+    @Override
+    public String render() {
```

Comment:
Be aware that last night I updated the branch with new commits, so rebase first :)